docs/build_docs.rst

```diff
@@ -13,7 +13,6 @@ Then, run:
 .. code-block:: bash
 
     cd docs
-    make clean
-    make html
+    DOCS_VERSION=test make clean versioned-html
 
-You can find the test build in ``_build/html/index.html``.
\ No newline at end of file
+You can find the test build in ``_build/html/index.html``.
```

Comment:
```suggestion
You can find the test build in ``_build/html/test/index.html``.
```

Reply:
Even better would be incorporating the changes like in:
https://github.com/E3SM-Project/polaris/blob/main/docs/developers_guide/building_docs.md

Obviously, the document is a different format, though.

Reply:
OK, just implemented this. Let me know what you think!